plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractBuildBootableJarMojo.java

```diff
@@ -285,6 +285,9 @@
     @Parameter(alias = "boot-logging-config", property = "wildfly.bootable.logging.config")
     private File bootLoggingConfig;
 
+    @Parameter(alias = "bootable-jar-build-artifacts", property = "wildfly.bootable.jar.build.artifacts", defaultValue = "bootable-jar-build-artifacts")
```

Comment:
@doychin , would you mind adding some javadoc to document this new parameter? Thank-you